Classes/Console/Command/Database/DatabaseUpdateSchemaCommand.php

```diff
@@ -76,6 +76,12 @@ protected function configure()
                     InputOption::VALUE_NONE,
                     'If set the updates are only collected and shown, but not executed'
                 ),
+                new InputOption(
+                    'raw',
+                    '',
+                    InputOption::VALUE_NONE,
+                    'If set the updates are only collected and written in a nice format'
```

Comment:
This description is misleading imho.

`--dry-run` only collects the updates, but does not execute them.
`--verbose` does not only output the tables which need changes, but also the sql satements
`--raw` in the current implementation always shows the raw SQL queries, no matter whether they are performed or not

I think the implementation is fine. Can you update the description to better describe what it does?

Reply:
I'll give it a spin. Thanks for your feedback.

Reply:
Is the better? 

`If set the updates statements are written to screen`

Reply:
How about `If set, only the SQL statements, that are required to update the schema, are printed`?